eng/versioning/version_client.txt

```diff
@@ -26,6 +26,8 @@ com.azure:azure-identity-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-messaging-eventhubs;5.1.0;5.2.0-beta.1
 com.azure:azure-messaging-eventhubs-checkpointstore-blob;1.1.0;1.2.0-beta.1
 com.azure:azure-messaging-servicebus;7.0.0-beta.2;7.0.0-beta.3
+com.azure:azure-data-schemaregistry;1.0.0-beta.1;1.0.0-beta.1
```

Comment:
nit: please put these in alphabetical order, they should be immediately after com.azure:azure-data-appconfiguration